### DISCUSSION TITLE
Use title field in yaml header as the default display in wikilinks

### DISCUSSION THREAD

**Discussion by damionx7** [damionx7/Zettel-Notes-Documentation#180] [Feature Requests]
[Peter A]: A wikilink has a file name and a label.  [[file_name|MyFileLabel]].  MyFileLabel displayed and when selected it opens file_name.  If the label is missing [[file_name]] then file_name is displayed as the label.  Instead I am proposing that instead of showing file_name, it should use the title field in the linked file for the display.

**Comment by irmod** [comment]
Confused...

**Reply by roke-julian-lockhart** [reply]
> https://github.com/damionx7/Zettel-Notes-Documentation/discussions/180#discussioncomment-11067849

@irmod, that's understandable - it's not particularly well phrased. The OP requests that WikiLink-style hyperlinks without labels render, from the YAML preamble of the linked file, its `title`-key's value, instead of the file name.
